client/src/pages/classes/[company_id]/list/index.tsx

```diff
@@ -103,8 +103,8 @@ const ClassList: FC = () => {
                 <Pagination
                   maxPages={5}
                   totalPages={Math.floor(

```

Comment:
we can simplify the logic by rounding up the quotient

```suggestion
                 totalPages={Math.ceil(numberOfUsers / limiter)}

```